tests/src/test/java/alluxio/client/rest/JobMasterRestApiTest.java

```diff
@@ -31,6 +35,15 @@
   private static final Map<String, String> NO_PARAMS = Maps.newHashMap();
   private LocalAlluxioJobCluster mJobCluster;
 
+  // TODO(chaomin): Rest API integration tests are only run in NOSASL mode now. Need to
+  // fix the test setup in SIMPLE mode.
+  @Rule
+  public LocalAlluxioClusterResource mResource = new LocalAlluxioClusterResource.Builder()
+      .setProperty(PropertyKey.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "false")
+      .setProperty(PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.NOSASL.getAuthName())
+      .setProperty(PropertyKey.USER_FILE_BUFFER_BYTES, "1KB")
+      .build();
+
```

Comment:
Does this test even need an Alluxio cluster?

Reply:
I tried removing it, but the test does not run. I think the cluster sets up all the journaling correctly, which the job cluster does not (yet).